target/linux/ramips/mt7621/config-6.6

```diff
@@ -317,3 +317,7 @@ CONFIG_ZLIB_INFLATE=y
 CONFIG_ZSTD_COMMON=y
 CONFIG_ZSTD_COMPRESS=y
 CONFIG_ZSTD_DECOMPRESS=y
+CONFIG_JFFS2_LZMA=y
```

Comment:
Please use `make kernel_menuconfig` to select these symbol.

Reply:
looks like CONFIG_JFFS2_LZMA is selected outside of config-6.6, good catch.